Doc/library/asyncio-sync.rst

```diff
@@ -285,11 +259,54 @@ Semaphore
 
 
 BoundedSemaphore
-^^^^^^^^^^^^^^^^
+----------------
 
 .. class:: BoundedSemaphore(value=1, \*, loop=None)
 
    A bounded semaphore implementation. Inherit from :class:`Semaphore`.
 
    This raises :exc:`ValueError` in :meth:`~Semaphore.release` if it would
    increase the value above the initial value.
+
+   Locks also support the :ref:`context management protocol <async-with-locks>`.
+
+   This class is :ref:`not thread safe <asyncio-multithreading>`.
+
+
+.. _async-with-locks:
+
+Using locks, conditions, and semaphores in the :keyword:`async with` statement
+------------------------------------------------------------------------------
+
+All of the objects provided by this module that have :meth:`acquire`
+and :meth:`release` methods can be used as context managers for a
```

Comment:
"can be used as context managers for a" -> "can be used in "async with" statements"